src/main.rs

```diff
@@ -252,7 +252,7 @@ pub fn main() {
         }
     } else {
         let mut input = String::new();
-        if let Err(why) = io::stdin().read_to_string(&mut input) {
+        if let Err(why) = io::stdin().lock().read_to_string(&mut input) {
```

Comment:
This line doesn't change anything, because that's already the internal implementation of `<Stdin as Read>::read_to_string`: <https://doc.rust-lang.org/src/std/io/stdio.rs.html#288-290>